app/views/content/subjects/engineers-teach-physics/_article.html.erb

```diff
@@ -115,14 +115,17 @@
     <h2 class="heading-s">Funding for teaching physics</h2>
 
     <p>
-     Scholarships of £26,000 or bursaries of £24,000 are available for trainee physics teachers. This is money that you do not have to pay back.
+     Scholarships of £26,000 or bursaries of £24,000 are available for trainee physics teachers starting their teacher training course in 2022.
+     </p>
+     <p>
+     Financial support for courses starting in 2023 will be announced by the end of the year.
```

Comment:
@gemmadallmandfe the message the FI team gave was that the amounts would be released 'soon', but I don't think that's particularly useful for users. I've put instead 'by the end of the year' - can we double check if this is accurate/if we have a more accurate timing for this?

Reply:
@EllieNodder the announcement is likely to be 'soon' but we don't have a clear date yet!